docker/README.md

```diff
@@ -0,0 +1,52 @@
+# gpusimilarity Docker Container
+
+The preferred method of using Docker is to just to download the publicly
+released pre-built docker images.  In addition, the enclosed Dockerfile can be
+built to produce an image capable of running GPUSimilarity.
+
+## Requirements
+* nvidia-docker (regular docker will not expose GPUs)
+* Newest versions of nvidia drivers
+* Recommended for production:  systemd / nginx
+
+## How to use pre-built container
+Both releases and master are always built and uploaded publicly already.  You
+can access them via:
+For master:
+```
+docker pull klorton/gpusimilarity:latest
+```
+
+For releases:
+
+```
+docker pull klorton/gpusimilarity:v1.0
+```
+
+The underlying requirements will change very little, so after the initial
+build/pull new docker layers will only require a few megabytes for
+gpusimilarity changes.
+
+## How to build (recommended only for development)
+From the docker directory:
+```
+docker build -t gpusim:internal .
+```
+
+## How to use to build a fingerprint file
+TODO
```

Comment:
FWIW, here's an adapted version of the command I used while testing:
`docker run -v /scratch/gpusim:/data -it gpusim python3 /gpusimilarity/bld/python/gpusim_createdb.py /data/zinc_all_clean.smi.gz /data/zinc_all_clean.fsim`
That's using my local tag/build and different paths, but does work

Reply:
This is great - thanks for providing!